airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -684,7 +684,6 @@ def conn(self) -> BaseAwsConnection:
         else:
             return self.get_resource_type(region_name=self.region_name)
 
-    @cached_property
```

Comment:
nit: you could have kept it as a (non-cached) property so that it can be used like `.conn` without parenthesis